docs/index.rst

```diff
@@ -4,7 +4,8 @@ OpenShift Build Service
 
 OSBS is a collection of tools, workflows and integration points that build and release layered container images.
 
-OSBS hooks into Koji_ with the help of the koji-containerbuild plugin, and
+OSBS hooks into Koji_ with the help of the `koji-containerbuild plugins
```

Comment:
If we are to refer to koji-container build as a set of plugins, perhaps we should do it everywhere. Namely, the koji-containerbuild repository as well.

Reply:
There are two plugins in the koji-containerbuild project: one for the CLI tool, and one for the builder. For the purpose of this sentence, OSBS uses the help of one of those plugins (the builder one). So I think singular makes sense here.

Reply:
To ease the acceptance of this particular pull request, I've deleted the final commit that pluralizes "plugins" here :) It's really a separate issue.